custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -116,12 +116,13 @@ def __init__(self, runtime):
                     "ensure there are no conflicts with other tasks downstream. Disable strict validation if you do "
                     "not want to use the default type schema."
                 )
-                use_default_type_schema = True
+                type_schema = read_model_metadata_yaml(
```

Comment:
ahhhh.  now i understand from comment in type_schema_validation,  please have a specific method for reading the default model_metadata.  if we only ever use this to get the typeschema, then please have the method specifically get the type schema.  this should be in `datarobot_drum.drum.model_metadata`

Reply:
yeah, makes sense

Reply:
assuming that this change is incoming, I'm going to approve now.  all else looks good